keyboards/melgeek/z70ultra/info.json

```diff
@@ -1,13 +1,7 @@
 {
     "keyboard_name": "Z70Ultra",
-    "manufacturer": "MelGeek",
     "url": "",
     "maintainer": "melgeek001365",
-    "usb": {
```

Comment:
Please revert.  USB info is being moved from the config.h to info.json.  

If needed, don't specify the PID, and just set the PID at the revision level.